content/conditions/hand-foot-and-mouth-disease/main-content.md

```diff
@@ -1,40 +1,72 @@
+The symptoms are usually the same in adults and children, but they can be much worse in adults.   
+
+It’s possible to get hand, foot and mouth disease more than once. 
+
 If you’re unsure it’s hand, foot and mouth disease, look at other
-[childhood rashes.](/symptoms/rashes-in-babies-and-children)
+[childhood rashes](https://beta.nhs.uk/symptoms/rashes-in-babies-and-children).
 
-## How to treat hand, foot and mouth disease at home
+Hand, foot and mouth disease has nothing to do with foot and mouth disease that affects farm animals.
 
-* drink fluids to avoid dehydration (avoid acidic drinks, like fruit juice)
-* eat soft foods like soup
-* take paracetamol or ibuprofen to help ease a sore throat and bring
-  down a temperature
 
-You can buy mouth ulcer gels, sprays, and mouthwashes to ease the
-  blisters. Always check they are suitable for children and give the
-  correct dose.
+## How to treat hand, foot and mouth disease yourself
+
+You can’t take antibiotics or medicines to cure hand, foot and mouth disease, it has to run it’s course.
+It usually gets better in 7 to 10 days.
+
+To help with the symptoms:
+
+- drink fluids to prevent dehydration (avoid acidic drinks such as fruit juice)
+- eat soft foods like soup (avoid hot and spicy foods) 
+- take paracetamol or ibuprofen to help ease a sore throat and bring down a temperature
+
+
+## Your pharmacist can help with hand, foot and mouth disease
+
+Your pharmacist can advise you on treatments, such as mouth ulcer gels, sprays, and mouthwashes to relieve pain.
+They can also tell you which ones are suitable for children.
 
-!!! attention
-## See a GP if
 
-* your child isn’t better after a week
-* your child is dehydrated --- they’re not weeing as often as usual
-* you’re concerned about your child’s symptoms
-* you’re pregnant and get hand, foot and mouth disease
+!!! attention
+  ## See a GP if:
+  
+  - your symptoms don’t improve after 7 to 10 days
+  - your temperature is 38C or above
```

Comment:
Same here about the degree character? Is the content pattern to not include them now?